screwdriver.yaml

```diff
@@ -132,3 +132,54 @@ jobs:
       - install-deps:     *bcsc-install-deps
       - build-multi-arch: *bcsc-build-multi-arch
       - *teardown-inspect
+
+  publish-build-almalinux-8:
+    image: almalinux:8.7
+    sourcePaths: ["screwdriver.yaml", "build/almalinux-8/"]
+    requires: [~pr, ~commit]
+    environment:
+      CONTAINER_DIR: 'build/almalinux-8'
+      CONTAINER_IMAGE: 'docker.io/vespaengine/vespa-build-almalinux-8'
+      CONTAINER_VERSION: 'latest'
+    steps:
+      - *inspect
+      - install-deps: &bcsc-install-deps |
+          dnf config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
```

Comment:
This one seems to require `dnf install -y dnf-plugins-core` in almalinux containers.